src/modules/AutoinstSoftware.rb

```diff
@@ -30,8 +30,10 @@ def main
       Yast.import "Mode"
       Yast.import "Misc"
       Yast.import "Directory"
+      Yast.import "Y2ModuleConfig"
 
       Yast.include self, "autoinstall/io.rb"
+      Yast.include Yast::Logger
```

Comment:
this is not correct. `Yast.include` is special Yast include suitable only for special Yast include files. Please use common
```
class XXX
  include Yast::Logger

  def main
     ...
```
as Yast::Logger is pure ruby module.